docs/FeatureManagement/Spring.FeatureFlag.v1.0.0.schema.json

```diff
@@ -0,0 +1,89 @@
+{
```

Comment:
This is client schema. It should go under
AppConfiguration/docs/FeatureManagement/Clients/SpringCloud

@jimmyca15 do we have something similar for .net? What are your thoughts about where to put them?